covid_graphs/covid_graphs/scripts/server.py

```diff
@@ -104,6 +107,10 @@ def _create_prediction_apps(data_dir: Path, server: Flask):
     def covid19_single_predictions():
         return single_prediction_app.index()
 
+    @server.route("/covid19/predictions/country/")
```

Comment:
This is a strange endpoint name. We have `/all` and `/single`, and `/country` to me sounds like synonymous with `/single`. 
```suggestion
    @server.route("/covid19/predictions/automatic/")
```

Reply:
What do you think about `daily`?

Reply:
Sounds good